release/models/platform/openconfig-platform-pipeline-counters.yang

```diff
@@ -1181,6 +1188,130 @@ module openconfig-platform-pipeline-counters {
     }
   }
 
+  grouping control-plane-traffic-counters-state {
+    description
+      "Control plane traffic counter state grouping.";
+
+    leaf total-queued {
+      type oc-yang:counter64;
+      description
+        "This captures the aggregation of all counters where the switch has enqueued
+        traffic related to the control-plane.";
+    }
+
+    leaf total-queued-bytes {
+      type oc-yang:counter64;
+      description
+        "This captures the aggregation of all counters in bytes where the switch has
+        enqueued traffic related to the control-plane.";
+    }
+
+    leaf total-dropped {
+      type oc-yang:counter64;
+      description
+        "This captures the aggregation of all counters where the switch has dropped
```

Comment:
Would this apply to both queue drops and filter/ACL drops?

Reply:
Yes, this should capture all packet drops to the control plane whether they are due to policer, queue tail drop, ACL drop or a drop due to an error.